app/editor/src/features/login/LoginPanel.tsx

```diff
@@ -3,7 +3,9 @@ import { Button, Col, useKeycloakWrapper } from 'tno-core';
 export const LoginPanel: React.FC = () => {
   const keycloak = useKeycloakWrapper();
 
-  const isLocal = window.location.host.startsWith('localhost');
+  const isLocal =
+    window.location.host.startsWith('localhost') ||
+    window.location.host.startsWith('host.docker.internal');
```

Comment:
Firefox isn't liking the cookie if it comes from a different domain anymore.  Which requires using `host.docker.internal` instead of `localhost`.